tests/dummy/app/pods/docs/deploying/template.md

```diff
@@ -73,7 +74,7 @@ This will produce two files in your current directory: `deploy_key` (the private
 
 On GitHub, open the page for your repo and navigate to _Settings_ -> _Deploy keys_ (or just directly visit <u>https://github.com/**[user]**/**[repo]**/settings/keys)</u> and click "Add deploy key".
 
-Enter a name for your key and then paste the contents of your public key (`id_rsa.pub`) into the big textarea. Make sure you check the **Allow write access** box, then click "Add key" and you're all set.
+Enter a name for your key and then paste the contents of your public key (`deploy_key.pub`) into the big textarea. Make sure you check the **Allow write access** box, then click "Add key" and you're all set.
```

Comment:
I'm _pretty sure_ you meant to reference the deployment key that was just created, not the default `id_rsa` key. This change makes it consistent with the documentation on `ember-cli-deploy-git-ci`